brd-ios/breadwallet/src/Redesign/Scenes/Swap/Details/Views/OrderView.swift

```diff
@@ -12,28 +12,33 @@ import UIKit
 
 struct OrderConfiguration: Configurable {
     var title = LabelConfiguration(font: Fonts.caption, textColor: LightColors.Text.one, textAlignment: .center, numberOfLines: 1)
-    var value = LabelConfiguration(font: Fonts.Body.two, textColor: LightColors.Text.one, numberOfLines: 1, lineBreakMode: .byClipping)
-    var icon = BackgroundConfiguration(tintColor: LightColors.Text.one)
-    var valueContent = BackgroundConfiguration(tintColor: LightColors.Text.one,
-                                               border: .init(tintColor: LightColors.Text.one,
-                                                             borderWidth: BorderWidth.minimum.rawValue,
-                                                             cornerRadius: .fullRadius))
+    var valueSmall = LabelConfiguration(font: Fonts.Body.two, textColor: LightColors.Text.one, textAlignment: .center, numberOfLines: 1, lineBreakMode: .byTruncatingTail)
+    var valueFull = LabelConfiguration(font: Fonts.Body.two, textColor: LightColors.Link.two, textAlignment: .center, numberOfLines: 0)
```

Comment:
valueSmall/valueFull ... what is this?
kind of seems 2 me like we try to "squeeze" two separate configs into one?

would it make sense to prepare a Small and Big preset for this view?